Update Creating-a-database-schema-from-models.md

shelldonhu opened the pull request:
It seems that the code below is incorrect because I get a string like 'NaN mysql'.
`console.log('Loopback tables [' - lbTables - '] created in ', ds.adapter.name);`
Maybe it should be like this:
`console.log('Loopback tables [' + lbTables + '] created in ', ds.adapter.name);`

Alibis committed:
Update Creating-a-database-schema-from-models.md

It seems that the code below is incorrect because I get a string like 'NaN mysql'.
<code>console.log('Loopback tables [' - lbTables - '] created in ', ds.adapter.name);</code>
Maybe it should be like this:
<code>console.log('Loopback tables [' + lbTables + '] created in ', ds.adapter.name);</code>

slnode commented:
Can one of the admins verify this patch?

dhmlau commented:
@Alibis , could you please sign the CLA?  https://cla.strongloop.com/agreements/strongloop/loopback.io

Also, there's linting error in your commit messages.  Here are the errors: 

```
**  Linting commit logs
**
**  3 problems found:
**    5bb54a5 - Update Creating-a-database-schema-from-models.md: line 3 longer than 72 characters.
**    5bb54a5 - Update Creating-a-database-schema-from-models.md: line 4 longer than 72 characters.
**    5bb54a5 - Update Creating-a-database-schema-from-models.md: line 6 longer than 72 characters.
```

Alibis committed:
Update Creating-a-database-schema-from-models.md

I get a response like 'NaN mysql' when the code using '-'.
Instead of '-', using '+' works well.

shelldonhu commented:
@dhmlau , I've signed the CLA.
I don't know that commit message have such limitation, a line's characters can't be longer than 72 characters.
I've just made a new commit, is it OK?

shelldonhu commented:
Or should I drop this commit and make a new one?

dhmlau commented:
@slnode test please

dhmlau commented:
@slnode test please

dhmlau commented:
@Alibis , thanks for signing the CLA. 
Looks like we're still getting the same linting error on the commit messages.  It's a bit strange, because I thought you only have 3 lines of commit messages where the errors are complaining about line 3,4, and 6.  Could you please double check on your side? 

BTW. your PR is outdated, could you please rebase as well?  Thanks.

dhmlau commented:
@bschrammIBM , if I click the View button to review the content, it seems like the code snippet was off.  Just need your review to make sure it will look fine on the staging environment.  Thanks.

dhmlau commented:
Closing this in favor of https://github.com/strongloop/loopback.io/pull/566.
